src/Microsoft.DotNet.XHarness.TestRunners.Common/ApplicationEntryPoint.cs

```diff
@@ -5,6 +5,7 @@
 using System;
 using System.Collections.Generic;
 using System.IO;
+using System.Linq;
```

Comment:
it would be nice to avoid the System.Linq dependency

Reply:
Switched the properties on the internal `ApplicationOptions` to be `ICollection` instead of `IEnumerable` and removed the System.Linq dependency.